test/MTLSpec.hs

```diff
@@ -0,0 +1,143 @@
+{-# LANGUAGE OverloadedStrings   #-}
+{-# LANGUAGE TypeApplications    #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+module MTLSpec where
+
+import           Polysemy
+import           Polysemy.Reader
+import           Polysemy.Writer
+import           Polysemy.State
+import           Polysemy.Error
+import           Polysemy.MTL
+
+import qualified Data.Text                     as T
+import           Test.Hspec
+
+import           Control.Monad                  ( replicateM_
```

Comment:
Put these all on one line please.

Reply:
Brittany seems to be preventing this.  I'll qualify instead?

Reply:
I think `Control.Monad` is a common-enough thing to import entirely without a qualification.